generate.py

```diff
@@ -93,28 +94,39 @@ def fix(obj):
 shutil.rmtree('schemas/', ignore_errors=True)
 
 # Make somewhere to store original schemas
-original_schema_path = pathlib.Path('schemas/original/')
+original_schema_path = Path('schemas/original/')
 original_schema_path.mkdir(parents=True, exist_ok=True)
 
 # Get schemas
 for schema_path in SCHEMAS:
     response = requests.get(API_SCHEMA + schema_path)
     if response.ok:
-        schema = response.text
-        with original_schema_path.joinpath(schema_path + '.json').open('w') as f:
-            f.write(schema)
+        schema_resp = response.text
+        original_schema_path.joinpath(schema_path + '.json').write_text(schema_resp)
 
 # Make somewhere to generate fixed schemas
-fixed_schema_path = pathlib.Path('schemas/fixed')
+fixed_schema_path = Path('schemas/fixed')
 fixed_schema_path.mkdir(parents=True, exist_ok=True)
 
 # Create a dummy item.json for the fixed schemas
-with fixed_schema_path.joinpath('item.json').open('w') as f:
-    item = {"type": "object", "properties": {}, "description": "A dummy Item, because Toshl devs decided to not include item.json"}
-    json.dump(item, f, sort_keys=True, indent=4)
+dummies = [
+    "item.json",
+    "export.filters.json",
+    "export.format.json",
+    "export.resource.json"
+]
+for dummy_json in dummies:
+    item = {
+        "type": "object",
+        "properties": {},
+        "description": f"Dummy Item, because Toshl devs do not include {dummy_json}"
+    }
+    dummy_data = json.dumps(item, sort_keys=True, indent=4)
+    fixed_schema_path.joinpath(dummy_json).write_text(dummy_data)
```

Comment:
These extra references look like the appeared recently. Good find. Interestingly, `formats` and `resources` should just be arrays of strings, according to [their doco](https://developer.toshl.com/docs/exports/create/).